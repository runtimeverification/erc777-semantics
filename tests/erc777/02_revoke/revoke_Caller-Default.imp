test.preamble();

// Testing revokeOperator(Operator) when Operator = caller, default operator
// must revert

test.makeCaller(8);
test.isOperatorFor(8, 8);
test.revokeOperator(8);
test.isOperatorFor(8, 8);
