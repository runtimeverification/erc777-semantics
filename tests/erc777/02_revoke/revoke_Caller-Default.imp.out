Changing the caller to 8
8 is operator for 8
8 is revoking operator 8
REVERT
8 is operator for 8
